### PR TITLE
Update next lint section

### PR DESCRIPTION
**Pull request by ghost**
App directory is stable so you no longer need appDir.

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 172c2644c7692bfb551e126c1956d7c28ac97ccf

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 172c2644c7692bfb551e126c1956d7c28ac97ccf

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Review by delbaoliveira — DISMISSED** [review]
Thank you!

File: docs/02-app/02-api-reference/08-next-cli.mdx [hunk 1]
```diff
@@ -125,7 +125,7 @@ npx next start --keepAliveTimeout 70000
 
 ## Lint
 
-`next lint` runs ESLint for all files in the `pages/`, `app` (only if the experimental `appDir` feature is enabled), `components/`, `lib/`, and `src/` directories. It also
+`next lint` runs ESLint for all files in the `pages/`, `app`, `components/`, `lib/`, and `src/` directories. It also
```
Comment (styfle) [inline docs/02-app/02-api-reference/08-next-cli.mdx:128]:
```suggestion
`next lint` runs ESLint for all files in the `pages/`, `app/`, `components/`, `lib/`, and `src/` directories. It also
```

**Review by styfle — APPROVED** [review]
Thanks!

**Pull request merged**